stellarphot/settings/custom_widgets.py

```diff
@@ -1,6 +1,18 @@
 # Some settings require custom widgets to be displayed in the GUI. These are defined in
 # this module.
 
+# This workaround is for Python < 3.11. It is not needed in Python 3.11 and later.
```

Comment:
Couldn't we just require 3.11?  I think we opted to require 3.10 last summer, but at this point...  In any case, if you are wedded to supporting 3.10, keep this.

Reply:
We could require 3.11. I wasn't aware until I opened this PR that `StrEnum` was new in 3.11.